tests/test_identity_pool.py

```diff
@@ -332,6 +333,7 @@ def test_from_info_required_options_only(self, mock_init):
             subject_token_type=SUBJECT_TOKEN_TYPE,
             token_url=TOKEN_URL,
             service_account_impersonation_url=None,
+            service_account_impersonation_info={},
```

Comment:
Add a test where the service_account_impersonation_options is there

Reply:
Ditto